ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -1115,3 +1121,16 @@ export function useSpecificWidgetDef(widgetId: string) {
   }, [frontstageDef, widgetId]);
   return widgetDef;
 }
+
+function WidgetFallback() {
```

Comment:
This seems to be copied, can't we simply re-use the existing component?

Reply:
Added `export` to the already existing component and reused it here.